src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -977,3 +977,20 @@ void CascadiaSettings::_resolveDefaultProfile() const
     // Use the first profile as the new default.
     GlobalSettings().DefaultProfile(_allProfiles.GetAt(0).Guid());
 }
+
+void CascadiaSettings::_validateCorrectDefaultShellPaths() const
+{
+    for (auto profile : _allProfiles)
+    {
+        if (profile.Guid() == DEFAULT_COMMAND_PROMPT_GUID &&
+            profile.Commandline() == L"cmd.exe")
+        {
+            profile.Commandline(L"%SystemRoot%\\System32\\cmd.exe");

```

Comment:
should we call `ClearCommandline` instead, to reset it to the inherited value? this would be somewhat more ergonomic in the settings UI and somewhat less ergonomic in the JSON, but it won't require us to repeat ourselves in three places with the new commandlines